app/views/merchant_items/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>All Items</h1>
+
+<% @items.each do |item| %>
```

Comment:
So right here, instead of @items, you can do @merchant.items.

Reply:
Done.